ragna/ui/central_view.py

```diff
@@ -0,0 +1,681 @@
+                                                                box-shadow: 0px 0px 6px 0px rgba(0, 0, 0, 0.2);
+                                                                padding: 10px 10px 10px 15px;
+                                                            }
+                                                        
+                                                            :host input[type="text"]:focus { 
+                                                                box-shadow: 0px 0px 8px 0px rgba(0, 0, 0, 0.3);
+                                                            }
+
+                                                         """
+                    ],
+                )
+            ],
+            renderers=[
+                lambda txt: RagnaChatMessage.chat_entry_value_renderer(txt, role=None)
+            ],
+            message_params={
+                "show_reaction_icons": False,
+                "show_user": False,
+                "show_copy_icon": False,
+                "show_timestamp": False,
+                # the proper avatar for the assistant is not when replacing the default ChatMessage objects
+                # with RagnaChatMessage objects.
+                "avatar_lookup": lambda user: "👤" if user == "User" else None,
+            },
+        )
+
+        chat_interface._card.stylesheets += [
+            """ 
+                                             
+                                            :host { 
+                                                border:none !important;
+                                            }
+
+                                            .chat-feed-log {  
+                                                padding-right: 18%;
+                                                margin-left: 18% ;
+                                                padding-top:25px !important;
+                                                
+                                            }
+                                             
+                                            .chat-interface-input-container {
+                                                margin-left:19%;
+                                                margin-right:20%;
+                                                margin-bottom: 20px;
+                                            }
+
+
+                                            """
+        ]
+
+        # Trick to make the chat start from the bottom :
+        #  - move the spacer first, and not last
+        # chat_interface._composite.objects =  (
+        #     [pn.layout.spacer.VSpacer()] +
+        #     [
+        #     w
+        #     for w in chat_interface._composite.objects
+        #     if not isinstance(w, pn.layout.spacer.VSpacer)
+        # ])
+        # chat_interface._chat_log.stylesheets.append(
+        #     """ :host .chat-feed-log {
+        #             height: unset; max-height: 90%; }
+        #     """
+        # )
```

Comment:
We no longer want that, do we? If not, remove?

```suggestion
```

Reply:
We don't need it anymore, I kept it just in case, and for anyone curious to learn.